src/test/java/seedu/address/logic/parser/ListCommandParserTest.java

```diff
@@ -18,5 +26,7 @@ public void parse_invalidArgs_throwsParseException() {
                 MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
         assertParseFailure(parser, "a", String.format(
                 MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "%", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
```

Comment:
Similar comment regarding the insertion of white spaces :)

Reply:
Amended!